tsl/test/expected/compress_sparse_config.out

```diff
@@ -12,13 +12,40 @@ WARNING:  column type "timestamp without time zone" used for "ts" does not follo
  (1,public,test_settings,t)
 (1 row)
 
+-- defaults
```

Comment:
I suggest `\pset null $` or something, so that null and empty values are distinguishable in the test output.